src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx

```diff
@@ -413,21 +413,18 @@ For setting up alerts related to data ingest limits and query limits, see [Set a
 
 To see changes made to your account (for example, changes related to managing users), you can query [`NrAuditEvent`](/docs/insights/insights-data-sources/default-data/nrauditevent-event-data-query-examples).
 
-## Usage-related data types [#data-types]
+## Usage-related events and attributes [#data-types]
```

Comment:
@zuluecho9 I had thought we were trying to keep "events" as an internal term for the data dictionary and publicly call them data types. Why the switch?

Reply:
these are technically 'events' as we use them in the MELT word. in NRDB, there are events (like Transaction, and NrUsage, and NrConsumption, etc.) and then there is metric data (`Metric`) and there is trace data (`Span`) and there is log data (`Log`). Most things in NRDB are still 'events' as we define them for MELT.